gtda/diagrams/representations.py

```diff
@@ -179,51 +179,51 @@ def plot(self, Xt, sample=0, homology_dimensions=None):
             for dim in homology_dimensions:
                 if dim not in self.homology_dimensions_:
                     raise ValueError(
-                        f'All homology dimensions must be in '
-                        f'self.homology_dimensions_ which is '
-                        f'{self.homology_dimensions_}. {dim} is not.')
+                        f"All homology dimensions must be in "
+                        f"self.homology_dimensions_ which is "
+                        f"{self.homology_dimensions_}. {dim} is not.")
                 else:
                     homology_dimensions_arr = np.array(
                         self.homology_dimensions_)
                     ix = np.flatnonzero(homology_dimensions_arr == dim)[0]
                     _homology_dimensions.append((ix, dim))
 
-        layout = {
-            "xaxis1": {
-                "title": "Filtration parameter",
-                "side": "bottom",
-                "type": "linear",
-                "ticks": "outside",
-                "anchor": "x1",
-                "showline": True,
-                "zeroline": True,
-                "showexponent": "all",
-                "exponentformat": "e"
-            },
-            "yaxis1": {
-                "title": "Betti number",
-                "side": "left",
-                "type": "linear",
-                "ticks": "outside",
-                "anchor": "y1",
-                "showline": True,
-                "zeroline": True,
-                "showexponent": "all",
-                "exponentformat": "e"
-            },
-            "plot_bgcolor": "white"
-        }
+        layout = dict(
+            xaxis1=dict(
+                title="Filtration parameter",
+                side="bottom",
+                type="linear",
+                ticks="outside",
+                anchor="x1",
+                showline=True,
+                zeroline=True,
+                showexponent="all",
+                exponentformat="e"
+            ),
+            yaxis1=dict(
+                title="Betti number",
+                side="left",
+                type="linear",
+                ticks="outside",
+                anchor="y1",
+                showline=True,
+                zeroline=True,
+                showexponent="all",
+                exponentformat="e"
+            ),
+            plot_bgcolor="white"
+        )
```

Comment:
I don't know much about this. Could I ask why would we prefer `dict()` over `{}`?

Reply:
In several IDEs (like Pycharm), syntax colouring will be more helpful when using `dict()`, as it clearly separates the keywords from the values. Also, as far as I remember it is the way `plotly` documentation tends to prefer things done (for whatever reason).